.goreleaser.yml

```diff
@@ -0,0 +1,65 @@
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+builds:
+  - env:
+      - CGO_ENABLED=0
+    binary: os-agent
+    ldflags:
+      - -s -w
+      - -X main.version={{.Version}}
+    goos:
+      - linux
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - 5
+      - 7
+
+archives:
+  - replacements:
+      386: i386
+      amd64: x86_64
+      arm64: aarch64
+    wrap_in_directory: true
+    files:
```

Comment:
```suggestion
    wrap_in_directory: true
    name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ end }}'
    files:
```
If we do not add the version to the file, instructions become simpler, and you can have a direct link to the latest asset.

Reply:
I don't agree with that, as the goal should be publishing it in an apt repository.

Reply:
For example:

![image](https://user-images.githubusercontent.com/195327/125511165-531d010a-93e1-41de-a28b-79c1743b4282.png)